cpapp_backend/cpapp/services/agent.py

```
@@ -4369,7 +5010,7 @@ def _format_bureau_decision_response(self, bureau_decision: Dict[str, Any], sess
 
 1. Continue with this limit
 2. Continue with limit enhancement"""
-            elif status and status.upper() == "REJECTED":
+            elif status and status.upper() == "REJECTED" or lender_decision == "REJECTED":
```

Ruff (0.14.5)

Parenthesize `a and b` expressions when chaining `and` and `or` together, to make the precedence clear

Parenthesize the `and` subexpression

(RUF021)